muselog/__init__.py

```diff
@@ -43,6 +43,10 @@ def setup_logging(root_log_level: Optional[str] = None,
         console_handler = logging.StreamHandler()
         console_handler.setFormatter(logging.Formatter(fmt=console_handler_format or DEFAULT_LOG_FORMAT))
         root_logger.addHandler(console_handler)
+    
+        # Python's root_logger by default adds a streamhandler if none is specified at app initilization
```

Comment:
This should be pydoc, not a regular comment.

Reply:
Can you clarify what you mean by that @ian-axelrod? When I google `pydoc` it refers to a pyhton module for generating python documentation in HTML. Do you mean a module level docstring?

Reply:
looked at that just now, do I wrap this in a string doc comment?
e.g.

```
"""
Note on add_console_handler: Python's root_logger by default adds a streamhandler if none is specified at app initialization
 if this is present we want it tracked and taken out later, not to be sent to datadog 
(remove double entry as it doesn't have the JSON formatter for easy interpretation on Datadog)
"""
```

Reply:
I mean pydoc. For example,

```
    def __init__(self,
                 url: str,
                 connector: Optional[Callable[[str], IO]] = None):
        """Create a feed.

        :param url:                 The url where this feed is found.
        :param connector:           A function that knows how to connect to the feed url.
                                    If not specified, the `smart_open` library is used.
        """
```